src/components/MultiGesture.tsx

```diff
@@ -166,9 +166,18 @@ class MultiGesture extends React.Component<MultiGestureProps> {
         // initialize this.currentStart on the the first trigger of the move event
         // TODO: Why doesn't onPanResponderStart work?
         if (!this.currentStart) {
-          // ensure that disableScroll is false when starting in case it wasn't reset properly
-          // may be related to https://github.com/cybersemics/em/issues/1189
-          this.disableScroll = false
+          // Check if we're in the gesture zone before deciding whether to disable scrolling
+          // This ensures we only prevent scrolling in the gesture zone, but allow it elsewhere
+          const touchLocation = e.nativeEvent.touches?.[0] || e.nativeEvent
```

Comment:
The TS engine is telling me that `e.nativeEvent.touches` is always defined. Is there a reason for the `?.` optional chaining?

1. Is it possible that `e.nativeEvent.touches` is undefined, which would be contrary to the `GestureResponderEvent` type?

Or

2. Is it possible that the array is empty, in which case `e.nativeEvent.touches?.[0]` would be `undefined`, but the `?.` optional chaining doesn't affect anything.

Reply:
I appreciate the extra caution, but let's investigate if there are actual cases where it would be undefined (if the type definition is inaccurate). Otherwise, best to remove and simplify it 👍